projects/storefrontapp-e2e-cypress/cypress/e2e/regression/consignment-tracking/consignment-tracking.core-e2e.cy.ts

```diff
@@ -21,6 +27,7 @@ describe('consignment tracking', () => {
       cy.get('.close').click();
       cy.get('.cx-list')
         .next()
+        .first()
```

Comment:
Fixing the new restriction on 'within', force single subject.
tested, no side-effect.